tools/ansible/roles/dockerfile/templates/Dockerfile.j2

```diff
@@ -5,7 +5,7 @@
 ###
 
 # Build container
-FROM centos:8 as builder
+FROM {{ base_container | default('centos:8') }}
```

Comment:
I'd still think you'd want this to include the `as builder`, as other parts of the templated container file [expect it to be a staged build](https://github.com/ansible/awx/blob/devel/tools/ansible/roles/dockerfile/templates/Dockerfile.j2#L168).

Reply:
@jakemcdermott Good call. I'll fix that now.